packages/amqp/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@message-queue-toolkit/amqp",
-  "version": "12.0.1",
+  "version": "12.0.2",
```

Comment:
Not sure if `patch` or `minor`, the new parameter is there but it can't be used so I guess it is a patch